src/app/beer_garden/local_plugins/manager.py

```diff
@@ -696,6 +696,11 @@ def _process_args(plugin_config: Dict[str, Any], instance_name: str):
         if plugin_args is not None:
             process_args += plugin_args
 
+        if plugin_config.get("GROUPS"):
+            plugin_groups = plugin_config["GROUPS"]
+            if plugin_groups is not None:
+                process_args += ["--groups=" + arg for arg in plugin_groups]
+
```

Comment:
You specified `group` and `groups` in the config options but you only used `groups`. We should check if either is present, then append `group` to the `groups` list.

Reply:
I'm not entirely sure `group` is needed here as it works fine without it, but added it anyway. However, adding `group` to the `groups` list will cause duplicates.